reviews/utils/review_actions.py

```diff
@@ -78,6 +79,20 @@ def __str__(self):
         return mark_safe(self.text_with_link())
 
 
+class SetEmailCheckbox(ReviewAction):
+    def is_available(self):
+        user_groups = self.user.groups.values_list("name", flat=True)
+        if not settings.GROUP_SECRETARY in user_groups:
+            return False
+        return True
+
+    def action_url(self):
+        return reverse("reviews:update_email_checkbox", args=(self.review.pk,))
+
+    def description(self):
+        return _("Zet email waarschuwing")
```

Comment:
```suggestion
        return _("'Extra info via e-mail' waarschuwing toevoegen")
```
And translate it as:
"Add 'Extra info via e-mail' warning"